indico/modules/events/abstracts/__init__.py

```diff
@@ -111,8 +111,8 @@ def _extend_event_menu(sender, **kwargs):
     from indico.modules.events.layout.util import MenuEntryData
 
     def _is_visible(conf):
-        return (session.user and bool(get_user_abstracts(conf.as_event, session.user)) and
-                conf.has_feature('abstracts'))
+        return (conf.has_feature('abstracts') and session.user and
```

Comment:
`session.user / feature / get_user_abstracts` - the first one should be the cheapest one